tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj

```diff
@@ -120,7 +120,6 @@
     <Compile Include="EmittedIL\SerializableAttribute\SerializableAttribute.fs" />
     <Compile Include="EmittedIL\SeqExpressionStepping\SeqExpressionStepping.fs" />
     <Compile Include="EmittedIL\SeqExpressionTailCalls\SeqExpressionTailCalls.fs" />
-    <Compile Include="EmittedIL\SeqExpressionStepping\SeqExpressionStepping.fs" />
```

Comment:
Did you remove the right line.  it looks to me like you intended to remove: 
 <Compile Include="EmittedIL\SeqExpressionTailCalls\SeqExpressionTailCalls.fs" />
 
 Which does indeed appear duplicated.

Reply:
Oh, I didn't even see the other duplication (there was no build warning for it). But I did indeed intend to remove this line, which is duplicated as line 121. I'll push up the removal of the following one too.